pyrobosim_ros/pyrobosim_ros/ros_interface.py

```diff
@@ -138,6 +140,13 @@ def __init__(
             callback_group=ReentrantCallbackGroup(),
         )
 
+        self.reset_world_srv = self.create_service(
+            ResetWorld,
+            "reset_world",
+            self.reset_world_callback,
+            callback_group=ReentrantCallbackGroup(),
```

Comment:
Since we won't be resetting worlds concurrently, MutuallyExclusiveCallbackGroup is okay.